bgs/bgs.go

```diff
@@ -400,6 +401,18 @@ func (s *BGS) syncUserBlobs(ctx context.Context, pds *models.PDS, user bsutil.Ui
 	return nil
 }
 
+func handleFromDidDoc(doc *did.Document) (string, error) {
```

Comment:
Related to https://github.com/bluesky-social/indigo/issues/56, we should technically be checking that the AlsoKnownAs value starts with `at://`. That is, iterate through the list until we hit a URI starting with `at://` and take that one.